src/App.js

```diff
@@ -1,22 +1,24 @@
-import React, { Component } from 'react';
-import { Navbar, Button } from 'react-bootstrap';
-import './App.css';
+import * as React from 'react'
+import { Navbar, Button } from 'react-bootstrap'
+import './App.css'
 
-class App extends Component {
+class App extends React.Component {
   goTo(route) {
     this.props.history.replace(`/${route}`)
   }
 
   login() {
-    this.props.auth.login();
+    this.props.auth.login()
   }
 
   logout() {
-    this.props.auth.logout();
+    this.props.auth.logout()
   }
 
   render() {
-    const { isAuthenticated } = this.props.auth;
+    const { isAuthenticated } = this.props.auth
```

Comment:
I would suggest changing this at some point to pass the value of `isAuthenticated()` in directly as a prop instead of passing the `auth` object as a prop. That way, if the `auth` prop changes later, you won't have to update this component's logic to adjust. 

With my suggestions, this line would change to:
`const { isAuthenticated } = this.props`
and 
`isAuthenticated` would be a boolean, not a function.